src/node/desktop/test/core/xdg.test.ts

```diff
@@ -184,7 +184,17 @@ describe('Xdg', () => {
   describe('Misc helpers', () => {
     it('SHGetKnownFolderPath returns a string', () => {
       if (process.platform === 'win32') {
-        // TODO, actually check results?
+        const user = os.userInfo().username
```

Comment:
These aren't guaranteed to be at these locations (hence the need for an API), but should suffice on any machine we're likely to run these tests on.